amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -3760,3 +3904,98 @@ definitions:
           - SiteCollaborator
           - SiteContributor
           - SiteManager
+  EventsInfo:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/EventInfo'
+  EventInfo:
+    type: object
+    required:
+      - id
+      - name
+      - type
+    properties:
+      id:
+        type: string
```

Comment:
Missing descriptions for the properties.

Reply:
Same comment for all objects.

Reply:
Done